aea/configurations/base.py

```diff
@@ -1647,10 +1658,10 @@ def from_json(cls, obj: Dict):
         component_configurations = {}
         for config in obj.get("component_configurations", []):
             tmp = deepcopy(config)
-            name = tmp["name"]
-            author = tmp["author"]
-            version = tmp["version"]
-            type_ = tmp["type"]
+            name = tmp.pop("name")
```

Comment:
see above; `tmp` at the end of the iteration will contain only the configurable fields.